README.md

```diff
@@ -1,28 +1,57 @@
+<div align="center">
+
 # Dynamic Bones Stub
-  
-[![Generic badge](https://img.shields.io/badge/Unity-2019.4.31f1-informational.svg)](https://unity3d.com/unity/whats-new/2019.4.31)
-[![Generic badge](https://img.shields.io/badge/SDK-AvatarSDK3-informational.svg)](https://vrchat.com/home/download)
-[![Generic badge](https://img.shields.io/badge/License-MIT-informational.svg)](https://github.com/VRLabs/Dynamic-Bones-Stub/blob/main/LICENSE)
+
 [![Generic badge](https://img.shields.io/github/downloads/VRLabs/Dynamic-Bones-Stub/total?label=Downloads)](https://github.com/VRLabs/Dynamic-Bones-Stub/releases/latest)
+[![Generic badge](https://img.shields.io/badge/License-MIT-informational.svg)](https://github.com/VRLabs/Dynamic-Bones-Stub/blob/main/LICENSE)
+[![Generic badge](https://img.shields.io/badge/Unity-2019.4.31f1-lightblue.svg)](https://unity3d.com/unity/whats-new/2019.4.31)
+[![Generic badge](https://img.shields.io/badge/SDK-AvatarSDK3-lightblue.svg)](https://vrchat.com/home/download)
+
+[![Generic badge](https://img.shields.io/discord/706913824607043605?color=%237289da&label=DISCORD&logo=Discord&style=for-the-badge)](https://discord.vrlabs.dev/)
+[![Generic badge](https://img.shields.io/endpoint.svg?url=https%3A%2F%2Fshieldsio-patreon.vercel.app%2Fapi%3Fusername%3Dvrlabs%26type%3Dpatrons&style=for-the-badge)](https://patreon.vrlabs.dev/)
+
+Stub package for the [Dynamic Bones](https://assetstore.unity.com/packages/tools/animation/dynamic-bone-16743) components whitelisted by VRChat
+
+![Not Dynamic Bones](https://github.com/VRLabs/Dynamic-Bones-Stub/assets/76777936/1ace50c9-475f-41ac-9312-98105277b578)
 
-Stub package for the VRChat and CVR whitelisted Dynamic Bones components. This package strips off everything that is not serialized, and only serves to preserve the serialized information in imported prefabs/scenes.
+### ⬇️ [Download Latest Version](https://github.com/VRLabs/Dynamic-Bones-Stub/releases/latest)
+
+<!-- 
+### 📦 [Add to VRChat Creator Companion]() -->
+
+</div>
+
+---
 
 ## How it works
 
-By having the same GUID as original scripts, it makes the scripts automatically reference the stubs. By using the same variable names, the serialized information gets transferred properly.
+* VRChat copies all component data into duplicate ones when your avatar is loaded.
+* By employing identical variable names in the stubs, the serialized information seamlessly transfers to the duplicate component.
 
-## Credit
+## Additional notes
 
-[Dreadrith](https://github.com/Dreadrith)
+* You cannot use any of the Dynamic Bone components in the editor, as they lack executable code.
+* For testing in the editor, we highly recommend purchasing the full software from the [Unity Asset Store](https://assetstore.unity.com/packages/tools/animation/dynamic-bone-16743).
```

Comment:
Here i would add a point saying that nowadays you should be using Physbones and that Dynamic Bones should only be used for support on old avatars